SKILL.tsv

```diff
@@ -2269,34 +2269,34 @@ SKILL_20150317_002268	Monstrance: Follow enenmy
 SKILL_20150317_002269	Magic circle of [Monstrance] follows nearby monsters.
 SKILL_20150317_002270	xxMonstrance: Evasion Defense
 SKILL_20150317_002271	xxWhen allies are within [Monstrance] magic circle, evasion and Physical defense of target increase by 2% per attribute level.
-SKILL_20150317_002272	Blessing: Number of attacks
+SKILL_20150317_002272	$Blessing: Number of Attacks
 SKILL_20150317_002273	xxNumber of attacks of [Blessing] increases by 1 per level.
-SKILL_20150317_002274	Blessing: Additional buff
+SKILL_20150317_002274	$Blessing: Additional Buff
 SKILL_20150317_002275	xxNumber of targets of [Blessing] increases by 1 per Attribute level.
 SKILL_20150317_002276	xxSacrament: Dark Property resistance
 SKILL_20150317_002277	xxDark Resistance increases by 5 per attribute levvel while [Sacrament] is activated.
-SKILL_20150317_002278	Resurrection: HP recovery
+SKILL_20150317_002278	$Resurrection: HP Recovery
 SKILL_20150317_002279	A Character revived by[Resurrection] will recover 10% of max. HP per attribute level.
 SKILL_20150317_002280	Resurrection: Resurrection count
 SKILL_20150317_002281	xxNumber of targets revived by [Resurrection] increases by 1 per Attribute level.
-SKILL_20150317_002282	Blunt Mastery: Stun
+SKILL_20150317_002282	$Blunt Mastery: Stun
 SKILL_20150317_002283	xxEnemies attacked with [Strike] type basic attacks have a 2% chance per Attribute level of being Stunned.
-SKILL_20150317_002284	Krivis: Fire attribute resistance
+SKILL_20150317_002284	$Krivis: Fire Property Resistance
 SKILL_20150317_002285	xxCharacter's Fire Resistance increases by 5 per attribute level while Dark Resistance decreases by 3 per attribute level.
-SKILL_20150317_002286	Zalciai: Intelligence
+SKILL_20150317_002286	$Zalciai: Intelligence
 SKILL_20150317_002287	Effect of [Zalciai] to clerics is changed from STR to INT.
-SKILL_20150317_002288	Divine Stigma: intelligence
-SKILL_20150317_002289	Strength increases by 20% per attribute level while Intelligence decreases by 20% per level when using [Divine Stigma].
-SKILL_20150317_002290	Zaibas: Splash
+SKILL_20150317_002288	$Divine Stigma: Intelligence
+SKILL_20150317_002289	$Increases INT, but not STR when using [Divine Stigma] by 20% per level.
```

Comment:
Fair enough, the wording is still a bit odd. Does strength need to be mentioned at all since the attribute just increases is INT?

"Increases INT by 20% per level when using [Divine Stigma]."